### ISSUE TITLE
Cosmetics menu command that can be used externally.

### ISSUE THREAD

**Issue by demirium** [PluginBugs/Issues-CosmeticsCore#167]
### Terms

- [X] I'm using the very latest version of CosmeticsCore and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-CosmeticsCore/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://lonedev.gitbook.io/cosmeticscore/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

demirium

### Describe the solution you'd like

We can create a wardrobe, but some of the players say that they waste a lot of time previewing and sometimes skip even using the system because of that. 

Can you add a command like /cosmetics or /cosmetic that opens the cosmetics menu externally and when clicked, players can remove and wear the cosmetics?

In short: the cosmetics menu that opens when you enter the wardrobe can be opened and controlled by an external command without any cinematics or additional animations.

### Is your feature request related to a problem?

Nope

### Describe alternatives you've considered

The cosmetics menu that opens when you enter the wardrobe can be opened and controlled by an external command without any cinematics or additional animations.

**Comment by LoneDev6** [comment]
Duplicate of https://github.com/PluginBugs/Issues-CosmeticsCore/issues/49
